_src/images/misc/5f75b5e6e24b376ed9ab5c1e_programming-browser.svg

```diff
@@ -0,0 +1,20 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we come up with a more descriptive folder name than `misc` for these? Maybe which section of the site they will live in?

Reply:
Changed to `card-icons` because these icons are used exclusively for cards.